### ISSUE TITLE
improve instructions for indexing?

### ISSUE THREAD

**Issue by sam327** [algolia/algoliasearch-magento#264]
This is a request for some better details on how frequent I should be indexing. I want to make sure I'm indexing often enough to make sure that prices are up to date, but not so often that I'm hitting my API limit. 

I've had a couple one-day sales recently, and the sale prices have lingered in the search results after the sales ended. (Because the reindex was only being forced every other day)

Something like: 
* You should schedule a reindex of the main Search index every X days, more or less often depending on how regularly you add new products.
* The other indexes should be reindexed every Y days. 
* if you flush the image cache, you need to reindex the main Search index.
* When a sale starts (or ends), you need to reindex all the indexes to get the updated prices. 

Basically, it's not completely clear when the plugin updates things on its own vs when I need to force it to update with the latest data.

**Comment by maxiloc** [comment]
The is not one solution because it really depends on the usecase.
I think what we can add in the docs a "when is a full reindex needed" section

**Comment by maxiloc** [comment]
This is now part of #238. Closing this
